platform/ios/app/MBXViewController.m

```diff
@@ -301,6 +316,7 @@ - (void)parseFeaturesAddingCount:(NSUInteger)featuresCount
                 MGLPointAnnotation *annotation = [MGLPointAnnotation new];
                 annotation.coordinate = coordinate;
                 annotation.title = title;
+                annotation.subtitle = [NSString stringWithFormat:@"View: %i", useViews];
```

Comment:
Yeah, was thinking about that but trying to avoid overhead of a class. Probably a good idea, though.